pkgs/tools/text/mmdoc/default.nix

```
@@ -0,0 +1,67 @@
+{stdenv,
+fetchFromGitHub,
+makeWrapper,
+
+cmark-gfm,
+coreutils,
+findutils,
+gnugrep,
+gnused,
+jq,
+} :
+
+let
+
+  runtimePath = stdenv.lib.makeBinPath [
+    cmark-gfm
+    coreutils
+    findutils
+    gnugrep
+    gnused
+    jq
+  ];
+
+in
+
+stdenv.mkDerivation rec {
+  pname = "mmdoc";
+  version = "0.0.4";
+
+  src = fetchFromGitHub {
+    owner  = "ryantm";
+    repo   = "mmdoc";
+    rev    = version;
+    hash = "sha256-p6ndIvmY2gkqDpGGv21U7POaS5xKJLn4Zz4jYiqJ2WA=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p "$out/bin"
+    mkdir -p "$out/lib/mmdoc"
+    install "$src/bin/mmdoc" "$out/bin"
+    install "$src/lib/mmdoc/fuse.basic.min.js" "$out/lib/mmdoc"
+    install "$src/lib/mmdoc/highlight.pack.js" "$out/lib/mmdoc"
+    install "$src/lib/mmdoc/minimal.css" "$out/lib/mmdoc"
+    install "$src/lib/mmdoc/mono-blue.css" "$out/lib/mmdoc"
+    install "$src/lib/mmdoc/search.js" "$out/lib/mmdoc"
```

Just a bit shorter and less repetition:

```suggestion
    install -D "$src/bin/mmdoc" "$out/bin/mmdoc"
    install -D -t "$out/lib/mmdoc" \
       "$src/lib/mmdoc/"{fuse.basic.min.js,highlight.pack.js,minimal.css,mono-blue.css,search.js}
```

---

I should probably move this to some Makefile in the project once I learn how.

---

Moved it to a Makefile and used your 2-liner. https://github.com/ryantm/mmdoc/blob/main/Makefile thanks!